packages/@tinacms/graphql/src/database/index.ts

```diff
@@ -290,10 +290,12 @@ export class Database {
       }
       this.contentLevel = this.contentNamespace
         ? this.rootLevel
-            .sublevel('_content')
+            .sublevel('_content', SUBLEVEL_OPTIONS)
             .sublevel(this.contentNamespace, SUBLEVEL_OPTIONS)
             .sublevel(version, SUBLEVEL_OPTIONS)
-        : this.rootLevel.sublevel(version, SUBLEVEL_OPTIONS);
+        : this.rootLevel
+            .sublevel('_content', SUBLEVEL_OPTIONS)
+            .sublevel(version, SUBLEVEL_OPTIONS);
     }
```

Comment:
I think this change is technically breaking but should not impact tina cloud since versioning is used in that case